public/app/core/components/TransformersUI/GroupByTransformerEditor.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useMemo } from 'react';
+import {
+  DataTransformerID,
+  SelectableValue,
+  standardTransformers,
+  TransformerRegistyItem,
+  TransformerUIProps,
+  ReducerID,
+} from '@grafana/data';
+import { getAllFieldNamesFromDataFrames } from './OrganizeFieldsTransformerEditor';
+import { Select, StatsPicker, Button } from '@grafana/ui';
+import { selectors } from '@grafana/e2e-selectors';
+
+import { GroupByTransformerOptions } from '@grafana/data/src/transformations/transformers/groupBy';
+
+function FieldCalculationsSelector(props: any) {
+  const { fieldNameOptions, onDelete, onConfigChange, config } = props;
+
+  return (
+    <div className="gf-form-inline">
+      <div className="gf-form gf-form-spacing">
+        <div className="gf-form-label width-8">On field</div>
+        <Select
+          className="width-16"
+          placeholder="Field Name"
+          options={
+            config[0] === null ? fieldNameOptions : [{ label: config[0], value: config[0] }, ...fieldNameOptions]
+          }
+          value={config[0]}
+          onChange={value => {
+            if (value === null) {
+              onConfigChange([null, config[1]]);
+            } else {
+              onConfigChange([value.value || null, config[1]]);
+            }
+          }}
+          isClearable
+          menuPlacement="bottom"
+        />
+      </div>
+      <div className="gf-form gf-form--grow gf-form-spacing">
+        <div className="gf-form-label width-8" aria-label={selectors.components.Transforms.Reduce.calculationsLabel}>
+          Calculate
+        </div>
+        <StatsPicker
+          className="flex-grow-1"
+          placeholder="Choose Stat"
+          allowMultiple
+          stats={config[1]}
+          onChange={stats => {
+            onConfigChange([config[0], stats as ReducerID[]]);
+          }}
+          menuPlacement="bottom"
+        />
+      </div>
+      <div className="gf-form">
+        <Button icon="trash-alt" onClick={onDelete} style={{ height: '100%' }} size="sm" variant="secondary" />
```

Comment:
Are inline stile ok ? I haven't find any `gf-form-*` classes that fit the need.

Reply:
Not sure what your trying to do here, we have not had this need to for height 100% on buttons before

Reply:
will try to test

Reply:
@mckn suggested that the delete (trash) button should take full height of the row:

> ## Suggestions on UI updates:
> 
> Make sure that we have the proper spacing between the different components (4px). The remove button should also take full height of the row.
> 
> <img alt="Skärmavbild 2020-08-07 kl 11 09 15" width="1212" src="https://user-images.githubusercontent.com/172951/89630242-5ffc6e80-d89f-11ea-9e2e-069dd57de937.png">
>